server/lib/enrich/article.js

```diff
@@ -33,6 +33,14 @@ module.exports = exports = function article(content, format) {
 
 	content.hasGraphics = Boolean(content.contentStats && content.contentStats.graphics);
 
+	// Currently embeds can have more than one item for each picture - for different screen sizes
+	// We are assuming that canBeSyndicated is the same for all sizes of a picture
+	// We are asking if ALL Graphics can be syndicated, which means as long as at least one item
+	// can't be, the answer to this is 'no'
+	const atLeastOneGraphicCantBeShared = content.embeds && content.embeds.filter(embed => embed && embed.type.endsWith('Graphic')).some(item => item.canBeSyndicated !== 'yes');
```

Comment:
```suggestion
	const atLeastOneGraphicCantBeShared = content.embeds && content.embeds.some(embed => embed && embed.type.endsWith('Graphic') && embed.canBeSyndicated !== 'yes');
```
Can be refactored to.

Reply:
If you don't mind, I find the staggered `filter - some` much easier to parse - I don't know how others feel about this.

Reply:
Also, 

Since the type is defined as `http://www.ft.com/ontology/content/Graphic`.
We can use equality for comparison instead of `endsWith('Graphic')`

Maybe personal preference